examples/js/loaders/AWDLoader.js

```diff
@@ -454,7 +459,9 @@ THREE.AWDLoader = ( function () {
 				var url = this.readUTFBytes( data_len );
 				console.log( url );
 
-				asset = this.loadTexture( url );
+				asset = this.loadTexture(url);
```

Comment:
```js
asset = this.loadTexture( url );
```
You should run `modularize.js` since the example uses now the module version of `AWDLoader`. Otherwise you can't test your change...

Reply:
Running modularize... Done.